src/encoded/tests/test_types_antibody_lot.py

```diff
@@ -672,23 +524,25 @@ def test_encode4_tagged_ab_review_status(testapp,
                        {'characterizes': biosample_1['@id']})
     res = testapp.get(encode4_tag_antibody_lot['@id'] + '@@index-data')
     assert len(res.json['object']['used_by_biosample_characterizations']) == 5
-    assert len(res.json['object']['lot_reviews']) == 2
-    assert {
-        'biosample_term_id': 'UBERON:0000948',
-        'biosample_term_name': 'heart',
-        'detail': 'Fully characterized.',
-        'organisms': ['/organisms/human/'],
-        'status': 'characterized to standards',
-        'targets': ['/targets/gfp-human/'],
-    } in res.json['object']['lot_reviews']
-    assert {
-        'biosample_term_id': 'UBERON:0002107',
-        'biosample_term_name': 'liver',
-        'detail': 'Fully characterized with exemption.',
-        'organisms': ['/organisms/human/'],
-        'status': 'characterized to standards with exemption',
-        'targets': ['/targets/gfp-human/'],
-    } in res.json['object']['lot_reviews']
+    # TODO: will be addressed in a PYTEST-REFACTOR
```

Comment:
@gabdank Here is another todo

Reply:
in this file I see 2 different TODOs. 
on line 527
in your implementation the lot reviews are:

> [{'biosample_term_name': 'heart', 'biosample_term_id': 'UBERON:0000948', 'organisms': ['/organisms/human/'], 'targets': ['/targets/gfp-human/'], 'status': 'characterized to standards', 'detail': 'Fully characterized.'}]

while it should have been 

> [{'biosample_term_name': 'heart', 'biosample_term_id': 'UBERON:0000948', 'organisms': ['/organisms/human/'], 'targets': ['/targets/gfp-human/'], 'status': 'characterized to standards', 'detail': 'Fully characterized.'}, {'biosample_term_name': 'liver', 'biosample_term_id': 'UBERON:0002107', 'organisms': ['/organisms/human/'], 'targets': ['/targets/gfp-human/'], 'status': 'characterized to standards with exemption', 'detail': 'Fully characterized with exemption.'}]

I think the reason being that you have changed "biosample_1" fixture to be heart, while it was liver originally.

 In general the follwoing looks wrong: 
```
@pytest.fixture
def biosample_1(testapp, lab, award, source, organism, heart):
    item = {
        'award': award['uuid'],
        'biosample_ontology': heart['uuid'],
        'lab': lab['uuid'],
        'organism': organism['uuid'],
        'source': source['uuid']
    }
    return testapp.post_json('/biosample', item, status=201).json['@graph'][0]

@pytest.fixture
def biosample_1_heart(testapp, lab, award, source, organism, liver):
    item = {
        'award': award['uuid'],
        'biosample_ontology': liver['uuid'],
        'lab': lab['uuid'],
        'organism': organism['uuid'],
        'source': source['uuid']
    }
    return testapp.post_json('/biosample', item, status=201).json['@graph'][0]

```
on line 931
The inconsistency is also due to the biosample_1 change

Reply:
@gabdank This was prematurely resolved so I missed it. I just pushed in a fix and awaiting a CI test-run